### ISSUE TITLE
[WIIU] No Accelerometer Support for Wiimote/Nunchuk

### ISSUE THREAD

**Issue by CFTBL** [libretro/RetroArch#13159]
Retroarch 1.19.11 on Wii U recognizes Wiimote and Nunchuk controllers (their buttons, DPad, and thumbsticks work). However, Retroarch doesn’t seem to support the accelerometer in the Wiimote, nor the accelerometer in the Nunchuk (if a Nunchuk is connected), nor the gyro in the MotionPlus attachment (if a MotionPlus attachment is connected). These motion devices should be supported so they can be assigned to specific axis when setting up controls. Obviously, it should be done in such a way that there isn’t a significant performance impact.

**Comment by paulwelka** [comment]
Yeah, good thought (IR support). Specifically, the ability to map the Wii pointer would improve the emulation of "aim and shoot" or "aim and throw" retro games. I think the ability to map all three Wiimote accelerometer axis would improve the emulation of retro games that used a spinner dial or two-way joystick (that's a lot of games, namely breakout style games, driving games, vertical shooters, etc).

**Comment by Ploggy** [comment]
Alot of missed potential when it comes to Wiimote utilisation in WiiU Retroarch.

**Comment by paulwelka** [comment]
Not sure if this helps, but if anyone is thinking about adding Wiimote accelerometer support, the interface and data reporting are pretty well documented, here: http://wiibrew.org/wiki/Wiimote
